packages/eslint-plugin/docs/rules/no-useless-constructor.md

```diff
@@ -0,0 +1,70 @@
+# Disallow unnecessary constructors (no-useless-constructor)
+
+ES2015 provides a default class constructor if one is not specified. As such, it is unnecessary to provide an empty constructor or one that simply delegates into its parent class, as in the following examples:
+
+```js
+class A {
+  constructor() {}
+}
+
+class A extends B {
+  constructor(value) {
+    super(value);
+  }
+}
+```
+
+## Rule Details
+
+This rule flags class constructors that can be safely removed without changing how the class works.
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```js
+/*eslint no-useless-constructor: "error"*/
+
+class A {
+  constructor() {}
+}
+
+class A extends B {
+  constructor(...args) {
+    super(...args);
+  }
+}
+```
+
+Examples of **correct** code for this rule:
```

Comment:
would suggest adding an example using parameter properties considering that's the point of this overload!